cmd/importverifier/importverifier.go

```diff
@@ -124,7 +124,7 @@ func (i *ImportRestriction) forbiddenImportsFor(pkg Package) []string {
 	forbiddenImportSet := map[string]struct{}{}
 	imports := pkg.Imports
 	if !i.ExcludeTests {
-		imports = append(imports, append(pkg.TestImports, pkg.XTestImports...)...)
+		imports = append(append(append([]string(nil), imports...), pkg.TestImports...), pkg.XTestImports...)
```

Comment:
It'll be more obvious if you could just create the `imports` copy two lines above where we careate `imports` variable.